policies/stable-release-updates.md

```diff
@@ -0,0 +1,81 @@
+Stable Release Update Policy
+============================
+
+This policy covers allowed changes on stable release branches.
+
+Definitions
+-----------
+
+A **stable release** is a patch release from an existing supported minor
+release branch.
+
+A **public interface** is any function, structure or macro declared in a public
+header file.
+
+A **bug fix** is a fix of functionality of the libraries, modules,
+applications, or the build system that (all or any items might apply):
+
+- makes the functionality to conform with the existing end-user documentation
+  where such a change does not break existing common usage
+- makes the end-user documentation to conform with the existing behavior of the
+  software
+
+A **bug fix** is also a fix for an unexpected behavior (not explicitly
+documented in one way or another) of the software when such unexpected behavior
+is a security vulnerability.
+
+A **bug fix** is also a fix for an unexpected behavior of the following kinds
+even if such behavior is not a security vulnerability:
+
+- a memory leak
+- a crash
+- a hang/deadlock or a race condition
+- an out of bounds read or write
+- an uninitialized value read
+- a memory use after free
+- C language undefined behavior
+
+A **bug fix** is also a build failure or test failure fix for platforms that
+are noted as supported for the stated release.
+
+A **bug fix** might also be a fix for an unexpected behavior (not explicitly
+documented in one way or another) where the implementation does not conform to
+existing public specifications that we claim to conform to. Where the
+documentation explicitly contradicts the specifications the documented behavior
+is what matters.
+
+A **functionality change** is **not**:
+
+- performance enhancement
+- memory usage reduction
+- replacement implementation of algorithms
+- refactoring
+
+An **end-user documentation** is not:
+- comments in the code (including public header files)
+- internal documentation (doc/internal)
+
+Changes allowed in stable releases
+----------------------------------
+
+No API or ABI breaking changes are allowed in a minor or patch release.
+
+Only bug fixes including fixes for security issues and end-user documentation
```

Comment:
Might be better written as: _Only bug fixes, fixes for security issues and end-user documentation_